scala.wake

```diff
@@ -9,6 +9,8 @@ def bloopInstall = "scala"
 def ivyCache = "ivycache"
 # Python wrapper script for invoking bloop
 def bloopWake = source "{here}/bloop_wake"
+# Where we expect to find wit/scalaplugin.py
+def scalaPluginPath = "wit/lib"
```

Comment:
If I'm reading this correctly, then I believe this is making an assumption about the relative path to the wit package from the workspace. Is there a way to make this more explicit and robust?

Reply:
That is correct. I guess I could add a wake file to wit that just declares the root directory. Does that make sense or can you think of something better?

Reply:
I like that idea, and I feel like it would be something useful to standardize on and possibly even build in to the Wake language. Would this possibly work well with idea of Wake packages/namespaces, @terpstra?

In the short term, I don't think we necessarily need to make a change to Wit before we can merge this in, but I'd like to see us take steps towards reducing the amount of implicit assumptions in these Wakefiles.

Reply:
Yeah I totally agree, I'm going to merge this as is so it can be bumped now, but language support or at least a standard pattern of how to declare where a package resides is the better way to do this.  Part of why I added `def scalaPluginPath` was just to make it more obvious that api-scala-sifive is assuming this, previously that assumption was embedded deeper in the code.